Packs/Akamai_SIEM/ReleaseNotes/1_2_10.md

```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### Akamai WAF Modeling Rule
+
+- Updated the Akamai WAF Modeling Rule modeling rule to support Native Collector.
```

Comment:
Updated the **Akamai WAF Modeling Rule** modeling rule to support the native collector.